### DISCUSSION TITLE
any Discord or Telegram Filecoin Channels? official, unofficial... but non scam?

### DISCUSSION THREAD

**Discussion by lherfel** [filecoin-project/community#562] [General]
any Discord or Telegram Filecoin Channels? official, unofficial... but non scam?

Thanks

**Comment by web3jenks** [comment]
Hi Lee, for official filecoin communities please stick to the ones on:
https://filecoin.io/

[image: Logo]

Jenks Guo

Developer Advocate

Filecoin Foundation & Filecoin Foundation for the Decentralized Web

✆: (+61) 04 225 902 59 🦘

Fil.org <https://fil.org> // FFDWeb.org <http://ffdweb.org> // Twitter
<https://twitter.com/JenksGuo> // LinkedIn
<https://www.linkedin.com/in/jenks-guo-58257438/> // Book a Meeting
<http://calendly.com/jenks-guo> // Telegram <https://t.me/jenksguo> //
Whatsapp <https://wa.me/+61422590259> // Element
***@***.***:matrix.org>
